### PR TITLE
Logica storage

### PR DESCRIPTION
**Pull request by darlison-alves**
Decorator para executar função com os dados do event em execução....

### CODE REVIEW

File: tsconfig.json [hunk 1]
```diff
@@ -10,8 +10,8 @@
         "sourceMap": true,
         "declaration": true
     },
-    "include": ["lib/**/*.ts", "tests/**/*.ts"],
-    "exclude": ["node_modules", "dist"],
+    "include": ["lib/**/*.ts"],
+    "exclude": ["node_modules", "dist", "tests/**/*.ts"],
```
Comment (iagosrodrigues) [inline tsconfig.json]:
```suggestion
    "exclude": ["node_modules", "dist", "tests"],
```

File: lib/decorator/StorageDecorator.ts [hunk 2]
```diff
@@ -0,0 +1,44 @@
+import Mercury, { JSONMessage } from '..';
```
Comment (iagosrodrigues) [inline lib/decorator/StorageDecorator.ts:1]:
```suggestion
import Mercury, { JSONMessage, Message } from '..';
```

File: lib/decorator/StorageDecorator.ts [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+import MessageEmitter from '../messageBus/MessageBusEventEmitter';
+import Message from '../message/Message';
```
Comment (iagosrodrigues) [inline lib/decorator/StorageDecorator.ts:3]:
```suggestion
```

File: lib/decorator/HandlerDecorator.ts [hunk 4]
```diff
@@ -43,6 +48,7 @@ const handler = (messageDescriptor: string = null, maxRetries: number = null): M
                 return result;
             } catch (e) {
                 if (message) {
+                    console.log(e.toString());
```
Comment (iagosrodrigues) [inline lib/decorator/HandlerDecorator.ts]:
```suggestion
```

**Review by iagosrodrigues — COMMENTED** [review]
I wrote some changes that might be interesting.

**Comment by cl3dson** [comment]
Darlinson, can you provide a example of how we use this decorator in the README?

**Pull request closed without merging**